extensions/oidc/deployment/src/main/resources/dev-ui/qwc-oidc-provider.js

```diff
@@ -1010,22 +1023,29 @@ export class QwcOidcProvider extends QwcHotReloadElement {
             const state = QwcOidcProvider._getQueryParameter('state');
             QwcOidcProvider._exchangeCodeForTokens(code, state, jsonRpc, onUpdateDone);
         } else {
-            // logged out
-
-            propertiesState.hideImplicitLoggedIn = true;
-            propertiesState.userName = null;
-
-            if (QwcOidcProvider._isErrorInUrl()) {
+            QwcOidcProvider._checkSessionCookie(jsonRpc, () => {
```

Comment:
@michalvavrik I don't think it is correct. Quarkus will never have a session cookie in a default case where `application-type=service`. It can only make sense for `web-app` applications

Reply:
@sberyozkin I didn't really test changes in DEV UI, just tried to use it. This changes doesn't come from me, but I'll have a look and try to fix your suggestion.

Reply:
Thanks @michalvavrik. When we use a `Login into SPA` option, Quarkus is not involved in the code flow, it is only a receiver of bearer access tokens. Our DevUI SPA manages the whole thing - and therefore there is no session cookie here. 

I think what @FroMage did there can be very useful for the `web-app`  application - which is when Quarkus itself manages  it

Reply:
I have tried it and indeed couldn't find a scenario with `service` where `loggedin` was called. Then I set authentication required for a root path and both hybrid and web-app application types are working - when you arrive to the OIDC provider card page, you already have tokens available. So I adapted JS to only attempt to retrieve tokens using session cookie when it's hybrid or web-app. Done.

Reply:
Done.